app/cdap/services/react/customHooks/useFeatureFlag.ts

```diff
@@ -28,7 +28,7 @@ const useFeatureFlag = (name: keyof IFeatureFlags | string, defaultResult: boole
     return defaultResult;
   }
 
-  return features[name] === 'true';
+  return features[name] === true || features[name] === 'true';
```

Comment:
the feature flags always come back as strings - did you add one in the ui config for this as a boolean? the temp feature flags should be what they actually come back as so this shouldn't be necessary.